### ISSUE TITLE
Additional rights option

### ISSUE THREAD

**Issue by kjrawson** [CollegeOfTheHolyCross/dta_sufia#81]
Hi, Steven! 
We are processing a small collection (in German!) that has a CC rights statement that they'd like us to include. It's not just a boilerplate one, but provides the specific attribution information as well. Could you add an additional way to add rights information? Maybe a repeated field with type-ahead? Or perhaps an option in the drop down menu to fill in text? After attending the Digital Commonwealth copyright seminar a few weeks ago, I am thinking more about how we might add additional statements to get more specific when we can.
Thank you!

**Comment by scande3** [comment]
In terms of the rights statement, we may need to do a call to discuss that at some point. A free text entry is dangerous.... but I could understand the issue less than I potentially should. For example, do you just want some added text in addition to the standard options? (So an additional field to the machine readable rights to expand upon the selected option with more text)? Have you looked at http://rightsstatements.org/en/ for standardization for a larger list btw? (The BPL will be moving to using those in the future).

**Comment by kjrawson** [comment]
Yes, that's what I am thinking––added text in addition to the standard options. That way, we could force one of the dropdowns, and an additional field could be optionally used if there are special cases where there are rights statements that our partners would like us to include. I have seen the additional statements and I am leaning toward incorporating a few:

- In copyright
- Copyright not evaluated
- Copyright undetermined
- No known copyright

The other statements are too specific for our uses––I wouldn't want to get too much into the business of specifying particular instances where usage would be okay because our permission agreement doesn't currently capture those nuances. And I could see cases when we would use each of the statements above.

Do you still want to have a call about this or is the direction making more sense?

**Comment by scande3** [comment]
Most of it makes sense and can take a stab at it for tonight to have up for tomorrow.

**Comment by scande3** [comment]
An addition field for free text rights has been added that has an autocomplete element. This is commit 0243d5887aafda97372efbaa16688a6e819614e6 .

Not sure how one might incorporate the list of rights into the site. (ie. if it becomes part of the rights dropdown or not). But let me know what needs to be adjusted from here or if the new field misses the mark.

Thanks!

**Comment by kjrawson** [comment]
The free text field works great––thanks!

For the list of rights: I was wondering whether it would be detrimental to change language in the rights field now that we have so many records with the other statements? I am happy to follow DC in moving towards the rightsstatements.org standardized versions, but I didn't know whether that would cause any problems on implementation. Two of them map easily onto our current options:

- In copyright = All rights reserved
- No known copyright =  No known restrictions on use

The others are somewhat different:
- Copyright not evaluated
- Copyright undetermined
- Contact host institution for more information [This could almost always pair with "In copyright"]

**Comment by kjrawson** [comment]
Will be wrapped up in new interface
